### PR TITLE
pixie: migrate to boneyard

### PR DESCRIPTION
**Pull request by JCount**
Created with `brew boneyard-formula-pr` because build is broken due to outdated zlib, last release was 19 May 2009.

### CODE REVIEW

**Comment by JCount** [comment]
for reference here is a snippet of the failure:
```Make
...
In file included from rendererContext.cpp:77:
./ribOut.h:261:36: error: no matching function for call to 'gzwrite'
        if (outputCompressed)   gzwrite(outFile,scratch,l);
                                                 ^~~~~~~
.../MacOSX10.12.sdk/usr/include/zlib.h:1351:21: note: candidate function not viable: \
no known conversion from 'FILE *' (aka '__sFILE *') to 'gzFile' (aka 'gzFile_s *') \
for 1st argument
ZEXTERN int ZEXPORT gzwrite OF((gzFile file,
                    ^
/bin/sh ../../libtool --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I. -I../.. -I.. \
-I/usr/local/Cellar/openexr/2.2.0/include/OpenEXR -I/usr/local/Cellar/ilmbase/2.2.0/include/OpenEXR \
-g -O2 -c -o rendererFiles.lo rendererFiles.cpp
2 errors generated.
make[3]: *** [rendererContext.lo] Error 1
...
```

File: tap_migrations.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
+  "pixie": "homebrew/boneyard"
```
Comment (MikeMcQuaid) [inline tap_migrations.json]: Needs a comma at the end of this line

**Comment by MikeMcQuaid** [comment]
> @MikeMcQuaid ping you on this, it is low priority, but I think it can be killed with 🔥. The developer has not maintained it at all since 2009 and has said as much; furthermore it is completely broken and in x11 so it doesn't even have still functional bottles, and again, x11. 😸

@JCount 👍 to boneyarding from me.

File: tap_migrations.json (hunk 1 above)
Comment (JCount) [inline tap_migrations.json]: oops, sorry about that @MikeMcQuaid

**Comment by MikeMcQuaid** [comment]
Thanks again @JCount!

**Pull request merged**